homeassistant/components/rfxtrx/strings.json

```diff
@@ -70,5 +70,16 @@
       "invalid_input_off_delay": "Invalid input for off delay",
       "unknown": "[%key:common::config_flow::error::unknown%]"
     }
+  },
+  "device_automation": {
+    "action_type": {
+      "send_status": "Trigger status: {subtype}",
```

Comment:
This is an action, maybe use imperative mood here?

Reply:
It's in imperative form here? it's not used in the the form "Status Trigger". In this order "Trigger Status" it's a command to trigger status.

But we could name it Send status update?

Reply:
I don't know the context well to judge if "Trigger Status" will be understood as a command to trigger status.
"Send status update" can't be misunderstood though.